docs/catalyst-client/overview.md

```diff
@@ -0,0 +1,28 @@
+# BigCommerce Catalyst Client
+
+Catalyst allows you to retrieve info from BigCommerce's APIs through the Catalyst Client.
+
+The client allows you to execute all queries and mutations available for the Storefront GraphQL API. You can also retrieve some information from BigCommerce's Management APIss.
+
+The client simplifies what you need to do to handle queries, requests, and responses. For example, you don't need to construct request URLs, configure request headers, or parse JSON responses. The client uses the channel ID and authorization tokens configured in the `.env` file by default.
+
+Include the following import statements:
+
+```ts
+import { client } from '~/client';
+import { graphql } from '~/client/graphql'; // A utility function for handling GraphQL queries
+```
+
+## Methods
+
+- `client.fetch()`: allows you to interact with BigCommerce's GraphQL Storefront API.
+
+- `client.fetchShippingZones()`: retrieves shipping zones for the BigCommerce store. It sends an HTTP GET request to the BigCommerce Management API, but abstracts the complexity of making a direct API request to fetch shipping zones.
```

Comment:
Ah thanks for the info. I'm inclined to have a separate `deprecated` section (rather than removing) so people using the old Catalyst repositories won't be confused. But I guess removing it might also be okay since Catalyst is new. @bookernath I wonder if you have a preferred direction?